src/Employer/UnitTests/Employer.Web/ViewModels/ApplicationReview/ApplicationReviewEditModelTests.cs

```diff
@@ -89,11 +96,39 @@ public void ShouldBeInvalid()
                 CandidateFeedback = "?$@#()\"\'\\!,+-=_:;.&€£*%/[] \\A-Z \a-z \0-9 your comments will be sent to the candidate."
             };
 
-            var validator = new ApplicationReviewEditModelValidator();
+            var validator = new ApplicationReviewEditModelValidator(MockProfanityListProvider);
 
             var result = validator.Validate(m);
 
             result.IsValid.Should().BeFalse();
         }
+
+        [Theory]
+        [InlineData(ApplicationReviewStatus.Unsuccessful, "Some candidate feedback bother")]
+        [InlineData(ApplicationReviewStatus.Unsuccessful, "dang Some candidate feedback")]
+        [InlineData(ApplicationReviewStatus.Unsuccessful, "Some candidate balderdash bother")]
+        [InlineData(ApplicationReviewStatus.Unsuccessful, "Some drat feedback bother")]
+        public void ShouldBeInvalid_ForProfanityWordsInFeedback(ApplicationReviewStatus outcome, string feedback)
+        {
+            var m = new ApplicationReviewEditModel
+            {
+                Outcome = outcome,
+                CandidateFeedback = feedback
+            };
+
+            var validator = new ApplicationReviewEditModelValidator(MockProfanityListProvider);
+            var result = validator.Validate(m);
+            result.IsValid.Should().BeFalse();
+            result.Errors.Count.Should().Be(1);
+            result.Errors[0].ErrorCode.Should().Be("617");
+        }
+    }
+
+    public class TestProfanityListProvider : IProfanityListProvider

```

Comment:
Use Mock Setup instead of concrete implementation

Reply:
👍